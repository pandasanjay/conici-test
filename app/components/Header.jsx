import React from "react";
import Icon from "../components/Icon";
import "./Header.scss";
const Header = ({ title }) => {
  return (
    <nav className="navbar navbar-light p-0 sh-navbar pt-4">
      <div className="p-2 ">
        <a className="sh-menu">
          <Icon iconname="menu-bar" />
        </a>
      </div>
      <div className="p-2 ">
        <a className="navbar-brand sh-title" href="#">
          {title}
        </a>
      </div>
      <div className="p-2 ">
        <a className="sh-location">
          <Icon iconname="location-uk" />
        </a>
      </div>
    </nav>
  );
};

export default Header;
