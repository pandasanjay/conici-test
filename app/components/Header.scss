@import "../var.scss";

.sh-navbar {
    border-bottom: 3px solid $primary-color;

    .sh-title {
        color: $header-title-color;
        font-family: $font-family
    }
}