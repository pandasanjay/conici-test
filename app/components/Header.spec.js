import React from "react";
import Header from "./Header";
import { shallow } from "enzyme";
describe("Header component", () => {
  let component;
  beforeEach(() => {
    component = shallow(<Header title="Smart Check-out" />);
  });
  it("should have match with snapshot", () => {
    expect(component).toMatchSnapshot();
  });
});
