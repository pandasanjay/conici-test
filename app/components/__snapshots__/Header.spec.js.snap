// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should have match with snapshot 1`] = `
<nav
  className="navbar navbar-light p-0 sh-navbar pt-4"
>
  <div
    className="p-2 pl-4"
  >
    <a
      className="sh-menu"
    >
      <Icon
        iconname="menu-bar"
      />
    </a>
  </div>
  <div
    className="p-2 "
  >
    <a
      className="navbar-brand sh-title"
      href="#"
    >
      Smart Check-out
    </a>
  </div>
  <div
    className="p-2 pr-4"
  >
    <a
      className="sh-location"
    >
      <Icon
        iconname="location-uk"
      />
    </a>
  </div>
</nav>
`;
